### PR TITLE
ePayments API documentation

### PR DESCRIPTION
**Pull request by ananthb86**
* How ePayment API works online
* How ePayment API works in store
* Features in ePayment API
  - Card payments
  - long living payments (merchant defined expiration time)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,19 @@ END_METADATA -->
...
 
-* Open the [Vipps landing page](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/vipps-landing-page).
-* Open the Vipps app on the user's mobile phone.
-* Cause a notification from the user's Vipps app.
-* Generate a QR code that a user can scan to send the request to their Vipps app.
+* [Vipps Wallet](/features/user-flow.md)
+* [Credit/Debit cards](/features/card-payments.md)
 
-You can also get payment details, get user info, capture payments, and refund payments.
+Merchant can
```
Comment (OFollan) [inline README.md:26]:
```suggestion
Merchant can 
* [create a/the payment](/create/README.md)
* [query the payment](/query/README.md)
* [capture/refund/cancel payment](/modify/README.md)
```
maybe? Don't know about the folder structure, but I feel `create -> query -> modify` can make sense.

File: features/card-payments.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: Card payments
+hide_table_of_contents: true
+pagination_next: null
+pagination_prev: APIs/epayment-api/getting-started
+---
+END_METADATA -->
+
+# Credit/Debit card payments
+
+Using epayment API merchants can also accept payments from users who do not have Vipps app by setting paymentMethod type as `CARD`.
```
Comment (OFollan) [inline features/card-payments.md:12]: ePayment API

File: features/customer-present-payments.md [hunk 3]
```diff
@@ -9,6 +9,6 @@ END_METADATA -->
 
 # Customer-present payments
 
-When using the ePayments API to process payments in a physical setting (e.g.,  from a Point Of Sale device, in a restaurant), the payment must be flagged with `"customerInteraction": "CUSTOMER_PRESENT"`.
+When using the ePayment API to process payments in a physical setting (e.g.,  from a Point Of Sale device, in a restaurant), the payment must be flagged with `"customerInteraction": "CUSTOMER_PRESENT"`.
```
Comment (OFollan) [inline features/customer-present-payments.md:12]:
```suggestion
When using the ePayment API to process payments in a physical setting (e.g. from a Point Of Sale device or in a restaurant), the payment must be flagged with `"customerInteraction": "CUSTOMER_PRESENT"`.
```

File: features/customer-present-payments.md [hunk 4]
```diff
@@ -9,6 +9,6 @@ END_METADATA -->
...
 
 `CUSTOMER_PRESENT` marking is required for compliance and reporting reasons.
```
Comment (OFollan) [inline features/customer-present-payments.md:14]: > for compliance and reporting reasons

I realize this is not part of this PR, but I am curious.
For whos compliance? For the Merchant to not break their merchant agreement? Or for Vipps? This seems a bit vague, would be nice to

Comment (ananthb86) [reply features/customer-present-payments.md:14]: These sections will be updated in the subsequent PRs with some context on what is to be considered CUSTOMER_PRESENT and CUSTOMER_NOT_PRESENT

File: create/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+flowchart TD
+    Payment --> Wallet[fa:fa-mobile paymentMethod: Wallet]
+    Wallet --> WEB_REDIRECT[fa:fa-desktop userFlow: WEB_REDIRECT]
+    Wallet --> PUSH_MESSAGE[fa:fa-bell userFlow: PUSH_MESSAGE]
+    Wallet --> QR[fa:fa-qrcode userFlow: QR]
+    Payment --> Card[fa:fa-credit-card paymentMethod:Card]
+    Card --> CARD_WEB_REDIRECT[fa:fa-desktop userFlow: WEB_REDIRECT]
+```
+
+`paymentMethod.type` in the request determines the type of payment. Allowed values are `CARD` and `WALLET`
```
Comment (rebekaburnett) [inline create/README.md:27]:
```suggestion
`paymentMethod.type` in the request determines the type of payment. Allowed values are `CARD` and `WALLET`.
```

File: features/long-living-payments.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+pagination_next: null
+pagination_prev: APIs/epayment-api/getting-started
+---
+END_METADATA -->
+
+The ePayment API supports long-living payments for paymentMethod type as `WALLET`, where the merchant can specify the expiration time when initiating the payment with POST:/epayment/v1/payments.
+
+The expiration time is set with the expiresAt parameter.
+
+See [Long living payments](https://vippsas.github.io/vipps-developer-docs/docs/vipps-solutions/long-expiry-time-for-payments-to-merchants) for more details.
```
Comment (rebekaburnett) [inline features/long-living-payments.md:14]:
```suggestion
See [Extended expiration for payments to merchants](https://vippsas.github.io/vipps-developer-docs/docs/vipps-solutions/long-expiry-time-for-payments-to-merchants) for more details.
```

Comment (rebekaburnett) [reply features/long-living-payments.md:14]: I think we should actually move the details they need from that solutions section and add links from the Solution section to here, for more details.

File: queries/README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 <!-- START_METADATA
 ---
-title: Queries
+title: Query Payment
 sidebar_label: Queries
 hide_table_of_contents: true
 pagination_next: null
```
Comment (rebekaburnett) [inline queries/README.md:6]: down below here, where it says "draft: true", just delete that line. It is what hides the page on docusaurus.

File: features/long-living-payments.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+<!-- START_METADATA
+---
+title: Long living payments
```
Comment (rebekaburnett) [inline features/long-living-payments.md:3]:
```suggestion
title: Long-living payments
```

File: features/long-living-payments.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+hide_table_of_contents: true
+pagination_next: null
+pagination_prev: APIs/epayment-api/getting-started
+---
+END_METADATA -->
+
```
Comment (rebekaburnett) [inline features/long-living-payments.md:9]:
```suggestion

# Long-living payments

```

File: vipps-epayment-api-how-it-works-in-store.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+<!-- START_COMMENT -->
+
+ℹ️ Please use the new documentation:
+[Vipps Technical Documentation](https://vippsas.github.io/vipps-developer-docs/).
+
+<!-- END_COMMENT -->
+
+Vipps in store is designed for stores that want to let customers pay easily and quickly from their own phone, without the need to touch the terminal. This page shows an example of how you can offer contactless payment to your customers by integrating Vipps in your POS system.
+
+It is also possible to use our simpler solution Vippsnummer, but then the amount must be entered manually at checkout, and there will be some follow-up with accounting and settlement.
```
Comment (rebekaburnett) [inline vipps-epayment-api-how-it-works-in-store.md:19]:
```suggestion
It is also possible to use our simpler solution [Vippsnummer](https://www.vipps.no/produkter-og-tjenester/bedrift/ta-betalt-i-butikk/ta-betalt-med-vipps/), but then the amount must be entered manually at checkout, and there will be some follow-up with accounting and settlement.
```

File: vipps-epayment-api-how-it-works-online.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+The payment is transferred to the merchant’s account. This may take 2-3 days depending on your bank.
+
+![Money](images/vipps-money.svg)
+
+
+## Pay with Credit/Debit cards
+
+If the users do not have Vipps profile or choose to pay with card they can select “Card” on the product page of a merchant’s website or app.
+
```
Comment (rebekaburnett) [inline vipps-epayment-api-how-it-works-online.md:66]: The following 2 images should be resized to be a bit smaller. :-)

Comment (ananthb86) [reply vipps-epayment-api-how-it-works-online.md:66]: We discussed, the images can be cropped later.

**Review by rebekaburnett — APPROVED** [review]
